connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/DB2IntegrationSuite.scala

```diff
@@ -45,15 +45,18 @@ class DB2IntegrationSuite extends DockerJDBCIntegrationV2Suite with V2JDBCTest {
     "scan with aggregate push-down: REGR_INTERCEPT with DISTINCT",
     "scan with aggregate push-down: REGR_SLOPE with DISTINCT",
     "scan with aggregate push-down: REGR_R2 with DISTINCT",
-    "scan with aggregate push-down: REGR_SXY with DISTINCT")
+    "scan with aggregate push-down: REGR_SXY with DISTINCT",
+    "simple timestamps roundtrip",
+    "simple timestamps pushdown")
 
   override val catalogName: String = "db2"
   override val namespaceOpt: Option[String] = Some("DB2INST1")
   override val db = new DB2DatabaseOnDocker
+  override def url: String = db.getJdbcUrl(dockerIp, externalPort)
```

Comment:
How about put `def url: String = db.getJdbcUrl(dockerIp, externalPort)` into `V2JDBCTest` and avoid the duplicate code ?

Reply:
Can't really because `db.getJdbcUrl(dockerIp, externalPort)` is dependent on `DockerJDBCIntegrationV2Suite`.

Reply:
Or we put it into `DockerJDBCIntegrationV2Suite` ?